loadtest/control/utils_test.go

```diff
@@ -36,7 +36,7 @@ func TestGetErrOrigin(t *testing.T) {
 }
 
 func TestEmulateUserTyping(t *testing.T) {
-	search := "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+	search := "this is long enough"
```

Comment:
Nothing to do with this PR. It was getting annoying to have to wait 20 seconds just for this.